core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -1290,7 +1291,12 @@ static void filterJarFile(Path resolvedDep, Path targetPath, Set<String> transfo
                 } else {
                     manifest = new Manifest();
                 }
-                try (JarOutputStream out = new JarOutputStream(Files.newOutputStream(targetPath), manifest)) {
+                try (JarOutputStream out = new JarOutputStream(Files.newOutputStream(targetPath))) {
+                    ZipEntry manifestEntry = new ZipEntry(JarFile.MANIFEST_NAME);
+                    // Set manifest time to epoch to always make the same jar
+                    manifestEntry.setTime(0);
```

Comment:
Why not call this before the try block and keep the manifest being passed as part of the `JarOuputStream constructor?

Reply:
Ignore this, I see it is set in the ZipEntry, not in the Manifest